examples/website-demos/social-messaging/src/components/SocialChannelList/SocialChannelList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { ChannelListMessengerProps } from 'stream-chat-react';
+
+import { SocialChannelListFooter } from '../SocialChannelList/SocialChannelListFooter';
+
+import './SocialChannelList.scss';
+
+type Props = ChannelListMessengerProps;
+
+export const SocialChannelList: React.FC<Props> = (props) => {
+  const { children } = props;
```

Comment:
will need loading (skeleton like current) and error states eventually, but all good for now